custom/setup.py

```diff
@@ -0,0 +1,53 @@
+import os
+import itertools
+
+from typing import Dict, List
+from setuptools import setup, find_packages
+
+ROOT_PATH = os.path.dirname(__file__)
+PKG_NAME = "mlservermllib"
+PKG_PATH = os.path.join(ROOT_PATH, PKG_NAME)
+
+
+def _load_version() -> str:
+    version = ""
+    version_path = os.path.join(PKG_PATH, "version.py")
+    with open(version_path) as fp:
+        version_module: Dict[str, str] = {}
+        exec(fp.read(), version_module)
+        version = version_module["__version__"]
+
+    return version
+
+
+def _extras() -> Dict[str, List[str]]:
+    extras = {
+        "mllib": ["pyspark==3.0.1"]
```

Comment:
Since this package only holds the `mlserver-mllib` extension, we can probably install it directly as a direct dependency rather than an extra. That way, users can just do:

```
pip install mlserver mlserver-mllib
```

Reply:
I'll add this in the README doc.